### PR TITLE
Add metrics for phase1

### PR DESCRIPTION
**Pull request by yontyon**
Add metrics for phase 1
See BC-2318.
In this PR we have added metrics for concord bft timers.

### CODE REVIEW

File: bftengine/src/bftengine/ReplicaForStateTransfer.cpp [hunk 1]
```diff
@@ -44,6 +45,7 @@ ReplicaForStateTransfer::ReplicaForStateTransfer(const ReplicaConfig &config,
 void ReplicaForStateTransfer::start() {
   stateTranTimer_ = TimersSingleton::getInstance().add(
       5s, Timers::Timer::RECURRING, [this](Timers::Handle h) { stateTransfer->onTimer(); });
+  metric_state_transfer_timer_.Get().Set(5000);
```
Comment (yuliasherman) [inline bftengine/src/bftengine/ReplicaForStateTransfer.cpp:48]: Please declare a variable for this value.

Comment (yontyon) [reply bftengine/src/bftengine/ReplicaForStateTransfer.cpp]: done

File: bftengine/src/bftengine/ReplicaForStateTransfer.cpp [hunk 2]
```diff
@@ -43,9 +43,10 @@ ReplicaForStateTransfer::ReplicaForStateTransfer(const ReplicaConfig &config,
 }
 
 void ReplicaForStateTransfer::start() {
+  std::chrono::milliseconds defaultTimeout = 5s;
```
Comment (yuliasherman) [inline bftengine/src/bftengine/ReplicaForStateTransfer.cpp:46]: Define as const

Comment (yontyon) [reply bftengine/src/bftengine/ReplicaForStateTransfer.cpp]: done

**Pull request merged**